requirements.txt

```diff
@@ -1,15 +1,12 @@
-jinja2~=2.11.2
-flask
-sparqlwrapper~=1.8.5
-pandas~=1.1.5
-pyyaml~=5.3.1
-toml~=0.10.1
-xlrd~=1.2.0
-pytest~=6.1.0
-numpy~=1.19.1
-tabulate~=0.8.7
-pytest-bdd~=4.0.1
-setuptools~=50.3.0
-Click~=7.1.2
-beautifulsoup4~=4.9.2
-rdflib~=5.0.0
\ No newline at end of file
+jinja2
+sparqlwrapper
+pandas
+pyyaml
+toml
```

Comment:
I recommend keeping the explicit versioning to avoid deterministic problems _(when a new dev environment is created, it might lead to unexpected errors because of different requirements installs)._

I really like this way of keeping track of then and easily accessing their documentation:
```
requests==2.24.0  # https://requests.readthedocs.io/en/master/
requests-toolbelt==0.9.1  # https://github.com/requests/toolbelt
rdflib==5.0.0  # https://rdflib.readthedocs.io/en/stable/
SPARQLWrapper==1.8.5  # SPARQLWrapper==1.8.5  #https://rdflib.dev/sparqlwrapper/

connexion[swagger-ui]==2.7.0  # https://github.com/zalando/connexion
Flask==1.1.2  # https://flask.palletsprojects.com/en/1.1.x/e
Flask-Wtf==0.14.3  # https://flask-wtf.readthedocs.io/en/stable/
```